.github/workflows/dapp-up-check.yaml

```
@@ -33,7 +33,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: '16'
+          node-version: '20'
```

This way we won't have to update it in multiple place next time we bump the node version.
```suggestion
          node-version-file: 'package.json'
```